app/services/workflow/instance/step/assign_person.rb

```diff
@@ -7,8 +7,10 @@ def initialize(step, person)
       end
 
       def run
-        @step.assignee_id = @person.id
-        @step.save!
+        # NOTE: if the step worktype was "only 1 assigner", we'd unassign first.
+        @step.assignments.find_or_create_by!(assignee: @person)
+        @step.assigned = true
+        @step.save
```

Comment:
Should we check if we should update the step's process' status to `in_progress`?